### PR TITLE
[build][pytorch] install missing packages for PT 1.5.0

### PR DESCRIPTION
**Pull request by tejaschumbalkar**
*Issue #, if available:*

## PR Checklist
- [x] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

## Pytest Marker Checklist
- [ ] (If applicable) I have added the marker `@pytest.mark.model("<model-type>")` to the new tests which I have added, to specify the Deep Learning model that is used in the test (use `"N/A"` if the test doesn't use a model)
- [ ] (If applicable) I have added the marker `@pytest.mark.integration("<feature-being-tested>")` to the new tests which I have added, to specify the feature that will be tested
- [ ] (If applicable) I have added the marker `@pytest.mark.multinode(<integer-num-nodes>)` to the new tests which I have added, to specify the number of nodes used on a multi-node test
- [ ] (If applicable) I have added the marker `@pytest.mark.processor(<"cpu"/"gpu"/"eia"/"neuron">)` to the new tests which I have added, if a test is specifically applicable to only one processor type

#### EIA/NEURON Checklist
* When creating a PR:
- [ ] I've modified `src/config/build_config.py` in my PR branch by setting `ENABLE_EI_MODE = True` or `ENABLE_NEURON_MODE = True`
* When PR is reviewed and ready to be merged:
- [ ] I've reverted the code change on the config file mentioned above
#### Benchmark Checklist
* When creating a PR:
- [ ] I've modified `src/config/test_config.py` in my PR branch by setting `ENABLE_BENCHMARK_DEV_MODE = True`
* When PR is reviewed and ready to be merged:
- [ ] I've reverted the code change on the config file mentioned above

## Reviewer Checklist
* For reviewer, before merging, please cross-check:
- [ ] I've verified the code change on the config file mentioned above has already been reverted

*Description:*
- There are some missing packages in DLC v1.1 release as compared to v1.0. This is due to the `sagemaker-containers ` package being deprecated and not installed in v1.1 release. Adding the missing packages back.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: pytorch/inference/docker/1.5.0/py3/Dockerfile.cpu [hunk 1]
```diff
@@ -99,7 +100,15 @@ RUN pip install \
...
-    "cryptography>=3.3.2"
+    "cryptography>=3.3.2" \
+    "paramiko==2.7.2" \
+    "Flask==1.1.2" \
+    "boto3==1.17.34" \
+    "click==7.1.2" \
+    "gevent==21.1.2" \
+    "gunicorn==20.0.4" \
+    "inotify-simple==1.3.5" \
+    "protobuf==3.15.6"
```
Comment (ti2ger92) [inline pytorch/inference/docker/1.5.0/py3/Dockerfile.cpu:111]: Why not use `protobuf==3.11.3` like this: https://paste.amazon.com/show/chumbalk/1616446911 ? also for other imports

Comment (tejaschumbalkar) [reply pytorch/inference/docker/1.5.0/py3/Dockerfile.cpu:111]: Is there a hard dependency on the package versions? we generally tend to install the latest package version available on pypi and conda.

File: pytorch/buildspec.yml [hunk 2]
```diff
@@ -1,7 +1,9 @@
 account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
```
Comment (tejaschumbalkar) [inline pytorch/buildspec.yml:1]: To be reverted on PR merge

**Review by saimidu — COMMENTED** [review]
As long as there are no safety test or functional test errors on this PR, I think the dockerfile changes are good, and can be merged in after all tests pass, and after reverting all temp changes.

APPROVED.

**Pull request merged**